### DISCUSSION TITLE
Do we need tuning for all different FOWTs but having the same rated power?

### DISCUSSION THREAD

**Discussion by IL-1** [NatLabRockies/ROSCO#250] [Q&A]
Dear all, 

I am responsible for some newly designed floaters for 15MW scale wind turbines.
I am just wondering if I can still use the same DISCON.IN file generated for UMaine Volturn-US floater.
In my understanding, ROSCO toolbox is capable of tuning floating feedback control gain based on the .fst and .yaml files.
But I am curious about its impact on the different floaters but having the same rated power.
 
Indeed, I applied the VolturnUS model's DISCON.IN file to a different floater (w/ different water depth and mooring layout), it returns quite different power production performance which is about 20MW not 15MW. But when I turn off the floating feedback control, it gives me reasonable power output.

If anybody can share insight, I will really appreciate it.

Regards,
IL

**Comment by dzalkind** [comment]
Hi,

In my experience, if the rotor is constant, but the platform is changed, the same controller can be used with reasonably similar performance.  I would have expected the power to be maintained, at least in a constant wind.  It might be worth checking the time series for instabilities in the controller/model.  Generally, FOWTs require floating feedback, unless it's a TLP.  Then, you can get away without it.

Here is some more information from a similar-sounding study: https://www.nrel.gov/docs/fy22osti/82038.pdf 

Best, Dan

**Reply by IL-1** [reply]
Thank you very much for your quick answer. The linked document is really helpful!
